spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/KafkaMessageChannelBinder.java

```diff
@@ -92,10 +93,8 @@
 import scala.collection.Seq;
 
 /**
- * A binder that uses Kafka as the underlying middleware. The general implementation mapping between XD concepts
- * and Kafka concepts is as follows:
- * A binder that uses Kafka as the underlying middleware.
- * The general implementation mapping between XD concepts and Kafka concepts is as follows:
+ * A {@link Binder} that uses Kafka as the underlying middleware.
```

Comment:
I guess we can probably remove this javadoc completely as we may not want `dataflow` references in SCSt too.